pkg/deployment/resources/pod_inspector.go

```diff
@@ -237,6 +237,19 @@ func (r *Resources) InspectPods(ctx context.Context) (util.Interval, error) {
 	allMembersReady := status.Members.AllMembersReady(spec.GetMode(), spec.Sync.IsEnabled())
 	status.Conditions.Update(api.ConditionTypeReady, allMembersReady, "", "")
 
+	if status.Members.Coordinators.AllFailed() {
```

Comment:
This expression seems to "soon".
Shouldn't it given the coordinators some time first?

Reply:
If they are failed, the operator will create a member replacement plan anyways. However, if all are failed, the execution of the plan will always fail because it can not connect to the cluster. So no, if all coordinators are failed, there is nothing we can do.